### PR TITLE
Move ARM storage account management cmdlets to a new project.

### PR DESCRIPTION
**Pull request by EmmaZhu**
Move ARM storage account management cmdlets to a new project.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@EmmaZhu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (emmazhu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@
 using Microsoft.Azure.Management.Storage;
 using System;
 
-namespace Microsoft.Azure.Commands.Compute
+namespace Microsoft.Azure.Commands.Management.Storage
```
Comment (huangpf) [inline src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs]: Any other options?

Comment (EmmaZhu) [inline src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs]: You mean other options for namespace? We decided to use this namespace after discussed with Mark and according to his suggestion..

Comment (huangpf) [inline src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs]: Yes, saw that. But what do you think of this namespace? It seems to me it's a bit confusing to put ```Commands``` and ```Management``` together, whereas the latter one usually means management library.

It seems to me the outlier is the ```Storage Data``` cmdlets, which shall be in a specialized namespace, not the ```Storage Management``` ones.

@markcowl Thoughts?

Comment (EmmaZhu) [inline src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs]: @huangpf , can you reply in the mail thread I just forward to you? We can discuss and make a decision in the mail thread.  
BTW, maybe next time you can raise your concern earlier before I sent the PR...:-)

Comment (huangpf) [inline src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/StorageManagementClient.cs]: As you may know most of the time it's clearer to directly look at the code. :-)

File: src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Azure.Subscriptions;
+using Microsoft.Azure.Test;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+
+namespace Microsoft.Azure.Commands.Management.Storage.Test.ScenarioTests
+{
+    public class TestController
+    {
+        bool testViaCsm = true; // Currently set to true, we will get this from Environment varialbe.
```
Comment (huangpf) [inline src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs]: This should not be needed anymore, BTW it's always ```true```.

File: src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        
+        public ResourceManagementClient ResourceManagementClient { get; private set; }
+
+        public SubscriptionClient SubscriptionClient { get; private set; }
+        
+        public AuthorizationManagementClient AuthorizationManagementClient { get; private set; }
+
+        public StorageManagementClient StorageClient { get; private set; }
+
+        public GalleryClient GalleryClient { get; private set; }
```
Comment (huangpf) [inline src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs]: I guess you don't really need this for SRP (and so does CRP). Just FYI.

**Comment by huangpf** [comment]
LGTM, with some minor comments that are just for your information. Thanks.

File: src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs (hunk 3 above)
Comment (EmmaZhu) [inline src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs]: I thought this too at the beginning. But the test framework will report an error without GalleryClient initialized. So I can just to put it here to resolve the issue.

File: src/ResourceManager/Storage/Commands.Management.Storage/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```
Comment (stankovski) [inline src/ResourceManager/Storage/Commands.Management.Storage/Properties/AssemblyInfo.cs]: Please add copyright headers to all cs files

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Pull request merged**